docs/ref/contrib/syndication.txt

```diff
@@ -961,6 +979,7 @@ They share this interface:
     * ``feed_copyright``
     * ``feed_guid``
     * ``ttl``
+    * ``stylesheet``
 
     Any extra keyword arguments you pass to ``__init__`` will be stored in
     ``self.feed`` for use with `custom feed generators`_.
```

Comment:
Further down here, we have the same "All parameters should be strings..." note where you might want to add "The ``stylesheet`` argument can be a string or an instance of :class:`Stylesheet`."

Reply:
Good catch, I've reused the same wording here as in the other file.